tests/ballerina-test/src/test/resources/test-src/documentation/function.bal

```diff
@@ -0,0 +1,18 @@
+documentation {
```

Comment:
Shall we add a test case for a scenario like below:
```ballerina
documentation { 
  Example of a string template:
    `string s = string `hello {{name}}`;`

  Example for an xml literal:
    `xml x = xml `<{{tagName}}>hello</{{tagName}}>`;`
}
```

Reply:
We agreed to use double back ticks for inline case. Added a test case for that.

```
documentation { 
  Example of a string template:
    ``string s = string `hello {{name}}`;``

  Example for an xml literal:
    ``xml x = xml `<{{tagName}}>hello</{{tagName}}>`;``
}
```